gusto/expressions.py

```diff
@@ -0,0 +1,202 @@
+"""
+Some thermodynamic expressions to help declutter the code.
+"""
+from firedrake import exp
+
+
+__all__ = ["theta_expr", "pi_expr", "pi_rho_expr", "pi_theta_expr", "p_expr", "T_expr", "rho_expr", "r_sat_expr", "Lv_expr", "theta_e_expr", "I_expr"]
+
+
+def theta_expr(parameters, T, p):
+    """
+    Returns an expression for dry potential temperature theta in K.
+
+    :arg parameters: an OutputParameters object.
```

Comment:
Are you sure these are `OutputParameters`?  And not `PhysicalParameters` or something like that?

Reply:
Ah, you're right, `CompressibleParameters` would be better!